ext/node/clippy.toml

```diff
@@ -1,6 +1,15 @@
 disallowed-methods = [
   { path = "std::env::current_dir", reason = "File system operations should be done using NodeFs trait" },
   { path = "std::path::Path::exists", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::canonicalize", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::is_dir", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::is_file", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::is_symlink", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::metadata", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::read_dir", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::read_link", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::symlink_metadata", reason = "File system operations should be done using NodeFs trait" },
+  { path = "std::path::Path::try_exists", reason = "File system operations should be done using NodeFs trait" },
```

Comment:
Do we need all of these for `PathBuf` too?

Reply:
Added